### ISSUE TITLE
Project > Detail > Resources: SortBy Resets FilterBy

### ISSUE THREAD

**Issue by thebigspoon** [Cadasta/ckanext-project#73]
Reproduce:
1. go to the project > detail > resource page
2. add some resources for a given project
3. select filterby "Parcel Resources"
4. select sortby "Name"

Expected:
list gets sorted

Actual:
Filter seems to be removed and I can see all the resources

**Comment by slindy23** [comment]
Is this one still open?

**Comment by thebigspoon** [comment]
yep, still a problem because we have multiple types of data sources we are filtering on
